BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -210,35 +210,11 @@ extension AttributedLabel {
             }
         }
 
-        /// The view needs to keep track of the current index for the accessibility rotor.
-        private var accessibilityLinkIndex = -1
-
-        /// These elements need to be retained by the view, and cannot be created inside the
-        /// `accessibilityCustomRotors` getter.
```

Comment:
This concern about the elements being retained is still valid. We cannot pass back unretained elements from the rotor search block. 

The trick here is  the  elements are  now retained in an array which is captured by the rotor's `itemSearchBlock`. 

Once VoiceOver releases the rotor these elements all get deallocated and then will be recreated the next time VoiceOver requests the rotor.